### ISSUE TITLE
[Tegumento] Replace/Reissue Cards

### ISSUE THREAD

**Issue by rachaelwoolen** [seedco/Workshop#8]
### What
Ability to replace and reissue cards directly from Teg. 

### Description
- Buttons on the card page that allow support to select `replace` or `reissue` card. 
- When `replace` or `reissue` is selected a drop down menu should appear allowing agent to select the reason for the replacement. 
- `Replace` button deactivates current immediately and generates an order for a new PAN.
- If lost or stolen is selected as replacement reason, a lost/stolen report should be generated to send to Visa. 
- `Reissue` button leaves leaves the current PAN active and generates an order for a new card with the same PAN. Card stays active until customer activates new card. 
- If available, include shipping options in Teg to allow for expedited shipping if someone has lost or broken their card and needs it asap. 

### User Story
Cards will inevitably need to be replaced for various reasons, when this happens, Support will need to order cards as quickly as possible. 

As a customer, if my card is lost or stolen, I'd like be able to receive a replacement with a new, secure PAN as soon as possible, so that I can have access to my business funds as quickly as possible. 

As a customer if my card has worn out, or is eaten by my dog (it happens more frequently than you'd think) I need a new card, but my PAN hasn't been compromised and I don't want to update all my online bills, etc. In this scenario I'd like to get a reissued card with the same PAN.

**Comment by benjisg** [comment]
I believe this one can now be closed out 😃

**Comment by rachaelwoolen** [comment]
Yep! I want to leave it open as a placeholder until we make a separate ticket just for reissues. If possible, we'd like to be able to reissue cards with the same PAN, but not a high priority.
